app/components/Chatbot/index.tsx

```diff
@@ -36,7 +35,6 @@ const QuestionInput = ({
 }: QuestionInputProps) => {
   const [question, setQuestion] = useState(initial || '')
   const {results, search, clear} = useSearch(1)
-  const clickDetectorRef = useOutsideOnClick(() => handleChange(''))
   const mobile = useIsMobile()
```

Comment:
@mruwnik I deleted this because it was annoying to have the question text get deleted whenever I clicked outside the input. Is that alright?

Reply:
yup. Check with @melissasamworth though